src/dcap_provider.cpp

```diff
@@ -54,12 +54,15 @@ static const std::map<std::string, std::string> default_values = {
 // New API version used to request PEM encoded CRLs
 constexpr char API_VERSION_LEGACY[] = "api-version=2018-10-01-preview";
 constexpr char API_VERSION[] = "api-version=2020-02-12-preview";
-
 static char DEFAULT_CERT_URL[] =
-    "https://global.acccache.azure.net/sgx/certificates";
+    "https://global.acccache.azure.net/sgx/certificates/";
 static std::string cert_base_url = DEFAULT_CERT_URL;
 
-static char DEFAULT_CLIENT_ID[] = "production_client";
+static char DEFAULT_THIM_AGENT_URL[] =
+    "http://169.254.169.254/metadata/THIM/sgx/getCerts?";

```

Comment:
Once THIM Agent updates its API to be more aligned with Intel, this should be updated. The final URL will be like: `http://169.254.169.254/metadata/THIM/sgx/certification/v3/pckcert?qeid={}&cpusvn={}&pcesvn={}&pceid={}&encrypted_ppid={}`

Reply:
Agreed! I coded this based on current documentation. Will update this